enarx-prstate

```diff
@@ -25,19 +26,31 @@ def handle(issue, pr, responsible):
     if needed:
         pr.add_to_assignees(*needed)
 
-# If there are any changes requested, the author is responsible.
+# Look through all PRs that have a changes requested review present.
 for issue in github.search_issues(f"org:enarx is:pr is:public is:open -is:draft review:changes_requested"):
```

Comment:
```suggestion
for issue in github.search_issues(f"org:enarx is:pr is:public is:open -is:draft"):
```

I think we just want all PRs, no?

Reply:
My thinking is that we can also assign reviewers automatically in this script.

Reply:
Later parts of the script assume it's working with a PR with a Changes Requested review, though that will probably change anyway.